hazelcast/src/test/java/com/hazelcast/jet/core/JobSerializerTest.java

```diff
@@ -151,13 +135,6 @@ public void when_serializerIsRegistered_then_itIsAvailableForLocalCacheSink() {
         );
     }
 
-    @Test
```

Comment:
Why was this removed? What's the behaviour change?

Reply:
It tests if `Value` can be serialized. With compact introduced it can. So it will not throw HazelcastSerializationException anymore. 
I was changing this `Value` with `Object` so that it will throw exception but we have decided not to exclude java classes from Compact Serialization. So I need to introduce a class that Compact can not serialize but it is serializable via a registered Serializer. 

But this was the case during the development. With Beta we have decided to make compact of by default, so this tests will pass again without any change. 

So I think I will revert similar tests for now to come up with a better way before finalizing this.

Reply:
Ok, thanks!

Reply:
fixed by f89791b